app/component/itinerary/RentalVehicleLinkContainer.js

```diff
@@ -0,0 +1,107 @@
+import { FormattedMessage, intlShape } from 'react-intl';
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import connectToStores from 'fluxible-addons-react/connectToStores';
+import { configShape, rentalVehicleShape } from '../../util/shapes';
+import {
+  getRentalNetworkConfig,
+  getRentalNetworkIcon,
+  getRentalVehicleLink,
+} from '../../util/vehicleRentalUtils';
+
+import withBreakpoint from '../../util/withBreakpoint';
+import Icon from '../Icon';
+import ExternalLink from '../ExternalLink';
+
+function RentalVehicleLinkContainer(
+  { isScooter, rentalVehicle, language, mobileReturn },
+  { config },
+) {
+  if (!rentalVehicle && !isScooter) {
```

Comment:
Is the component name correct? It seems to apply to scooters only. Should it be ScooterLinkContainer?

Also, I suggest making rentalVehicle  required prop and removing null check. Maybe isCooter is also irrelevant check here: it does not make sense to pass a prop isScooter = false and then return immediately null.

Reply:
This is the unfortunate result of a convoluted thought process where technically we could support vehicle types other than scooter but currently we only do support scooters. So, scooter link does make more sense right now.